shared/search/index.js

```diff
@@ -40,15 +30,27 @@ class Search extends Component<void, Props, void> {
 const connector: TypedConnector<TypedState, TypedDispatch<SearchActions>, OwnProps, Props> = new TypedConnector()
 
 export default connector.connect(
-  ({search: {searchHintText, searchPlatform, searchText, searchIcon, results}, config: {username}}, dispatch, ownProps) => ({
-    username: username || '',
-    searchHintText,
-    searchText,
-    searchIcon,
-    results,
-    showComingSoon: !flags.searchEnabled,
-    onClickResult: () => console.log('TODO'),
-    selectedService: searchPlatform,
-    onSearch: (term, platform) => { dispatch(search(term, platform)) },
-    onClickService: platform => { dispatch(selectPlatform(platform)) },
-  }))(Search)
+  ({search:
+     {searchHintText, searchPlatform, searchText, searchIcon, results, userForInfoPane, showUserGroup, selectedUsers},
+   config: {username}}, dispatch, ownProps) => ({
+     username: username || '',
+     searchHintText,
+     searchText,
+     searchIcon,
+     userForInfoPane,
+     results,
+     showComingSoon: !flags.searchEnabled,
+     onClickResult: user => { dispatch(addUserToGroup(user)) },
+     selectedService: searchPlatform,
+     onSearch: term => { dispatch(search(term, searchPlatform)); dispatch(hideUserGroup()) },
+     onClickService: platform => { searchPlatform !== platform && dispatch(selectPlatform(platform)) },
+     showUserGroup,
+     selectedUsers,
+     onRemoveUserFromGroup: user => { dispatch(removeUserFromGroup(user)) },
+     onClickUserInGroup: user => { dispatch(selectUserForInfo(user)) },
+     onAddAnotherUserToGroup: () => { dispatch(hideUserGroup()) },
+     onOpenPrivateGroupFolder: () => { console.log('TODO open private group') },
```

Comment:
why not dispatch the action here? we have the open in kbfs action already, and you know what the path is